modules/apps/layout/layout-content-page-editor-web-test/src/testIntegration/java/com/liferay/layout/content/page/editor/web/internal/portlet/action/test/AddSegmentsExperienceMVCActionCommandTest.java

```diff
@@ -162,24 +163,74 @@ public void testAddSegmentsExperiment() throws Exception {
 			_layout, defaultSegmentsExperienceId, segmentsExperienceId);
 	}
 
+	private void _addSegmentsExperienceData(
+			Layout layout,
+			LayoutPageTemplateStructureLocalService
+				layoutPageTemplateStructureLocalService,
+			LayoutPageTemplatesImporter layoutPageTemplatesImporter,
+			String pageElementJSON)
+		throws Exception {
+
+		LayoutPageTemplateStructure layoutPageTemplateStructure =
+			layoutPageTemplateStructureLocalService.
+				fetchLayoutPageTemplateStructure(
+					layout.getGroupId(), layout.getPlid());
+
+		String segmentsExperienceData =
+			layoutPageTemplateStructure.getDefaultSegmentsExperienceData();
+
+		LayoutStructure layoutStructure = LayoutStructure.of(
+			segmentsExperienceData);
+
+		layoutPageTemplatesImporter.importPageElement(
```

Comment:
@cgoncas, instead of branching in Util class, each tests imports the corresponding pageElement